src/tests/integration.md

```diff
@@ -0,0 +1,49 @@
+# Integration testing
+
+Rust tests integration with real-world code to catch regressions and make
+informed decisions about the evolution of the language.
+
+## Testing methods
+
+### Crater
+
+Crater is a tool which runs tests on many thousands of public projects. This
+tool has its own separate infrastructure for running, and is not run as part of
+CI. See the [Crater chapter](crater.md) for more details.
+
+### Cargo test
+
+`cargotest` is a small tool which runs `cargo test` on a few sample projects
+(such as `servo`, `ripgrep`, `tokei`, etc.).
+This runs as part of CI and ensures there aren't any significant regressions.
+
+> Example: `./x test src/tools/cargotest`
+
+### Integration builders
+
+Integration jobs build large open-source Rust projects that are used as
+regression tests in CI.
+
+Fuchsia is currently the only project used in this way. See the [Fuchsia
+chapter](fuchsia.md) for more details.
```

Comment:
We also have rust-for-linux

Reply:
Yes, that has changed since I wrote this :D